packages/rainbowkit/package.json

```diff
@@ -61,7 +61,7 @@
     "nock": "^13.4.0",
     "postcss": "^8.4.32",
     "react": "^18.3.0",
-    "vitest": "^0.33.0",
+    "vitest": "2.x",
```

Comment:
We should use a static version to prevent any errors from poping up in a minor release